html-templates/sbg/progress/section-term-reports/sbg.css.tpl

```diff
@@ -0,0 +1,41 @@
+{extends "progress/section-term-reports/sectionTermReport.css.tpl"}
+th, td.grade {
```

Comment:
code inside a template with `{extends}` that isn't within a `{block}` gets discarded